fiftyone/server/aggregations.py

```diff
@@ -132,7 +132,26 @@ async def aggregate_resolver(
         awaitable=True,
     )
 
-    slice_view = view if form.mixed and "" in form.paths else None
+    slice_view = (
+        fosv.get_view(
+            form.dataset,
+            view_name=form.view_name or None,
+            stages=form.view,
+            filters=form.filters,
+            extended_stages=form.extended_stages,
+            sample_filter=SampleFilter(
+                group=(
+                    GroupElementFilter(
+                        id=form.group_id, slice=form.slice, slices=[form.slice]
+                    )
+                    if not form.sample_ids
+                    else None
+                )
+            ),
```

Comment:
did you intend for slice view to not be awaitable like above view is?

Reply:
It should be awaitable. Thanks 👍